### PR TITLE
Fixes #139 : UI display the room id to share it 

### LINKED ISSUE #139: [Front] UI display roomId to endusers
End-users need to have access to room Ids as they need those IDs to join a room, we should display the roomIds somewhere.

### PR DESCRIPTION
**Pull request by yannis-mlgrn**
Fix #139 
I have added a button who when we click on it, it run a alert.

### CODE REVIEW

**Comment by Draeggon** [comment]
you can use something like this:
```js
share: function(currentRoom) { 
            id = this.currentRoom._id.$oid
            console.log(id);
            alert("id : " + id);
        }
```

**Comment by yannis-mlgrn** [comment]
In V1, I show the id in a alert. In  V2, I will show them in a popup

https://user-images.githubusercontent.com/48618655/120232263-7c64ff80-c253-11eb-9bc2-3abe03072597.mp4

**Comment by js-all** [comment]
laaaazzzyyyyyyyyyyyy

File: front/src/index.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <html>
     <head>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```
Comment (Draeggon) [inline front/src/index.html:3]: what is that ?

Comment (js-all) [reply front/src/index.html:3]: for icons but i would recomend [this](https://feathericons.com/) because open source

Comment (yannis-mlgrn) [reply front/src/index.html:3]: is a link to import googles icon than I have used

Comment (Draeggon) [reply front/src/index.html:3]: use the open source one's proposed by js-all

**Comment by yannis-mlgrn** [comment]
> laaaazzzyyyyyyyyyyyy

wait @js-all i will make a V2 haha

File: front/src/index.html (hunk 1 above)
Comment (Benvii) [reply front/src/index.html:3]: migth also be better to include the font in the project sources directly

File: front/src/js/activity_bar.js [hunk 2]
```diff
@@ -1,11 +1,19 @@
 const activityBar = {
     name: "activity-bar",
     props: ["currentRoom", "isWriting"],
+    methods:{
+        share: function(currentRoom) { 
+            id = this.currentRoom._id.$oid
```
Comment (Benvii) [inline front/src/js/activity_bar.js:6]: I think $iod was removed from the API responses it's now simply .id, @Draeggon can you confirm ?

**Review by Benvii — CHANGES_REQUESTED** [review]

File: front/src/js/activity_bar.js (hunk 2 above)
Comment (Draeggon) [reply front/src/js/activity_bar.js:6]: yep but @js-all will fix it in his pr that is coming soon

File: front/src/index.html (hunk 1 above)
Comment (yannis-mlgrn) [reply front/src/index.html:3]: okay

File: front/src/js/activity_bar.js (hunk 2 above)
Comment (Draeggon) [reply front/src/js/activity_bar.js:6]: and it would be
```js
this.currentRoom.id
```

Comment (yannis-mlgrn) [reply front/src/js/activity_bar.js:6]: yes in #151. The feature will come in front with #179

**Comment by yannis-mlgrn** [comment]
V2 :
https://user-images.githubusercontent.com/48618655/120548460-a99be400-c3f2-11eb-9ac1-18848554c0d2.mp4

**Comment by js-all** [comment]
ok you're almost there, just add
```css
your_popup {
position: fixed;
top: 1em;
left: 50%;
transform: translateX(-50%);
}
```

and you should be good

(also remove the background and border of the button)

**Comment by yannis-mlgrn** [comment]
now, it is necessary to center the popup but idk how to move it

**Comment by yannis-mlgrn** [comment]
thanks

**Comment by yannis-mlgrn** [comment]
poppup
You thinks is right or you want to put the popup on the center ?

**Comment by js-all** [comment]
you choose
to center it
```css
top:50%;
transform: translate(-50% -50%);
```

**Comment by yannis-mlgrn** [comment]
ok I center the popup but it not reponsive. @js-all, you know how to solve that ?

**Comment by js-all** [comment]
how did you center it ?

File: front/src/css/activity_bar.css [hunk 3]
```diff
@@ -1,3 +1,75 @@
+
+.share_popup {
+    position: fixed;
+    top:30%;
+    transform: translate(80%);
```
Comment (yannis-mlgrn) [inline front/src/css/activity_bar.css:5]: with these lines

**Comment by yannis-mlgrn** [comment]
@js-all with these lines :  
    `position: fixed;
    top:30%;
    transform: translate(80%);`

**Comment by yannis-mlgrn** [comment]
I have added a `z-index: 99;` because the timestamp appeared on the popup

**Comment by yannis-mlgrn** [comment]
For the share button, do you prefer a white or a dark grey button ?

**Comment by Draeggon** [comment]
> For the share button, do you prefer a white or a dark grey background color ?

dark grey is better

**Comment by yannis-mlgrn** [comment]
final screen :
screen

**Comment by Draeggon** [comment]
nice

File: front/src/index.html (hunk 1 above)
Comment (Draeggon) [reply front/src/index.html:3]: don't need that anymore since you put the images in icons file

File: front/src/js/activity_bar.js [hunk 4]
```diff
@@ -1,11 +1,30 @@
 const activityBar = {
     name: "activity-bar",
     props: ["currentRoom", "isWriting"],
+    methods:{
+        openPopup: function() {
```
Comment (Draeggon) [inline front/src/js/activity_bar.js:5]: it seem that with the vue v-show you don't need that function so delete it

**Review by Draeggon — CHANGES_REQUESTED** [review]
I haven't check the css but the other stuff is quiet good
only a few minor changes are needed

File: front/src/js/activity_bar.js (hunk 4 above)
Comment (yannis-mlgrn) [reply front/src/js/activity_bar.js:5]: yes i  have forgot to remove

Comment (Draeggon) [reply front/src/js/activity_bar.js:5]: happen a lot to me too :joy:

Comment (yannis-mlgrn) [reply front/src/js/activity_bar.js]: done here d4b541e

**Review by Draeggon — APPROVED** [review]

**Review by landerneRPi — APPROVED** [review]

**Review by Benvii — APPROVED** [review]
Ok

**Pull request merged**